src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -50,6 +50,28 @@
 
     private final ScheduledExecutorService executor = Executors.newScheduledThreadPool(Runtime.getRuntime().availableProcessors());
 
+    /**
+     * Filenames for which the template filename differs from the filename it should have in the repository.
+     */
+    // @formatter:off
```

Comment:
Why are you turning the formatter off here?
Does it mess up the indentation or something?

Reply:
Yes, it would place multiple entries of the map onto the same line. I think explicitly having one entry per line in this case makes it a lot clearer which patterns are replaced by what.

Reply:
I agree.
Using it should be fine in this case.